src/pages/plan-trip/plan-trip.html

```diff
@@ -30,7 +30,10 @@
         type="text" placeholder="Enter a destination" [(ngModel)]="request?.destination.name"/>
       </label>
     </ion-item>
-    <ion-item>
+    <ion-item *ngIf="isInternetExplorer">
+      <p style="font-style: italic;">Internet Explorer only supports finding buses leaving now.</p>
```

Comment:
Fair point. While we're on the subject, I realized that "leaving now" makes little sense without the context that there **was** another option.

I propose `Internet Explorer does not support searching the schedule on specific dates and times, and will find the only soonest available trip.`.